<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProbeGetCandidatePage</class>
 <widget class="QWizardPage" name="ProbeGetCandidatePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extract/Load Probe Candidates</string>
  </property>
  <property name="title">
   <string>Extract/Load Probe Candidates</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Extract probe candidates from target sequences or load user's probe candidates:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="pixmap">
           <pixmap resource="PRISE.qrc">:/new/prefix1/images/exclamation.bmp</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Note: If you load your own probe candidates, the program won't consider probe candidates extracted from target sequences</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_designByInput">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Design probes based on target and non-target sequences</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioButton_loadCands">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Load user's probe candidates</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_CandFilename"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_browseCandFile">
            <property name="text">
             <string>Browser</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder></zorder>
     <zorder>horizontalSpacer</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="PRISE.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_browseCandFile</sender>
   <signal>clicked()</signal>
   <receiver>ProbeGetCandidatePage</receiver>
   <slot>browseCandidatesFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseCandidatesFile()</slot>
 </slots>
</ui>
